### PR TITLE
Paper for JOSS

### PR DESCRIPTION
**Pull request by zonca**
See:
* author guidelines https://joss.readthedocs.io/en/latest/submitting.html
* review criteria https://joss.readthedocs.io/en/latest/review_criteria.html

Pushed by the author guidelines I improved the project introductory test in the docs and the README.

I created this branch in the `healpy` repo, so you can directly push modifications.

First please check your name (I am missing some middle initials) and add your affiliation, then send feedback on the text.

### CODE REVIEW

**Comment by DanielLenz** [comment]
Thanks for writing this up, @zonca ! I'll give it a thorough read over the weekend.

File: paper/paper.md [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+has proven to be an excellent mathematical framework to store map-domain data
+and efficiently compute their Spherical Harmonics Transform, whose Angular
+Power Spectrum is one the most powerful tools to understand the early Universe.
+See in the ![figure](healpix_grid.png) how a sphere is first split into 12 base
+pixels of equal area and whose centers are at the same latitude
+and then each is further subdivided to achieve higher and higher resolution.
+The ``HEALPix`` team provides FORTRAN, C++ and IDL implementations of the framework.
+
+``healpy`` is a wrapper to the multi-threaded ``HEALPix`` C++ library in Python, it implements
+a more user-friendly interface for all ``HEALPix`` functionalities.
```
Comment (mreineck) [inline paper/paper.md]: Can we agree to remove the "more"?

File: paper/paper.md [hunk 2]
```diff
@@ -0,0 +1,89 @@
...
+date: 1 February 2019
+bibliography: paper.bib
+---
+
+# Summary
+
+Recent experiments measuring the temperature and polarization of the Cosmic
+Microwave Background, like WMAP [@wmap13] and Planck [@planck18], produce all-sky maps at higher
+and higher resolution.
+Handling those datasets efficiently and study their statistical properties
```
Comment (DanielLenz) [inline paper/paper.md]: and studying their statistical properties

File: paper/paper.md [hunk 3]
```diff
@@ -0,0 +1,89 @@
...
+has proven to be an excellent mathematical framework to store map-domain data
+and efficiently compute their Spherical Harmonics Transform, whose Angular
+Power Spectrum is one the most powerful tools to understand the early Universe.
+See in the ![figure](healpix_grid.png) how a sphere is first split into 12 base
+pixels of equal area and whose centers are at the same latitude
+and then each is further subdivided to achieve higher and higher resolution.
+The ``HEALPix`` team provides FORTRAN, C++ and IDL implementations of the framework.
+
+``healpy`` is a wrapper to the multi-threaded ``HEALPix`` C++ library in Python, it implements
+a user-friendly interface for all ``HEALPix`` functionalities.
```
Comment (DanielLenz) [inline paper/paper.md]: functionalities, most importantly a fast nearest-neighbor search and the decomposition into Spherical Harmonics coefficients.

File: paper/paper.md [hunk 4]
```diff
@@ -0,0 +1,89 @@
...
+It also adds utilities to read and write maps, Spherical Harmonics coefficients and
+Power Spectrum values as FITS files based on ``astropy.io.fits``.
+Finally it provides extensive plotting functionality, i.e. Mollweide, Gnomonic and Cartographic
+projections based on Matplotlib.
+We also release a ``conda`` package on ``conda-forge`` and wheels on PyPI for Linux and MacOS. The packages
+that also bundle the ``HEALPix`` C++ library.
```
Comment (DanielLenz) [inline paper/paper.md]: These packages also bundle the

File: paper/paper.md [hunk 5]
```diff
@@ -0,0 +1,89 @@
...
+
+``healpy`` was designed to be used by professional cosmologists and students to analyze
+outputs of Cosmic Microwave Background experiments. WMAP, Planck
+and many other experiment in the field release their maps
+in ``HEALPix`` FITS format that can be accessed, visualized and analyzed with ``healpy``.
```
Comment (DanielLenz) [inline paper/paper.md]: visualized, and analysed

File: paper/paper.md [hunk 6]
```diff
@@ -0,0 +1,89 @@
...
+More recently, its usage spread to
+other branches of Astrophysics including galaxy surveys ([Sloan Digital
+Sky Survey depth maps](http://risa.stanford.edu/redmapper/)), Gamma-ray astronomy
+(Fermi [@deil17]) and gravitational waves measurements (LIGO [@singer16]).
```
Comment (DanielLenz) [inline paper/paper.md]: [), and gravitational

File: paper/paper.bib [hunk 7]
```diff
@@ -0,0 +1,84 @@
...
+      eid = {20},
+    pages = {20},
+      doi = {10.1088/0067-0049/208/2/20},
+   adsurl = {http://adsabs.harvard.edu/abs/2013ApJS..208...20B},
+  adsnote = {Provided by the SAO/NASA Astrophysics Data System}
+}
+
+
+@article{planck18,
+      author         = "Akrami, Y. and others",
```
Comment (DanielLenz) [inline paper/paper.bib]: Should we simply update this to "Planck Collaboration"?

File: paper/paper.md [hunk 8]
```diff
@@ -0,0 +1,89 @@
...
+Recent experiments measuring the temperature and polarization of the Cosmic
+Microwave Background, like WMAP [@wmap13] and Planck [@planck18], produce all-sky maps at higher
+and higher resolution.
+Handling those datasets efficiently and study their statistical properties
+requires a discretization scheme on the sphere.
+The Hierarchical Equal Area isoLatitude Pixelation ([``HEALPix``](https://healpix.sourceforge.io), [@gorski05]) scheme
+has proven to be an excellent mathematical framework to store map-domain data
+and efficiently compute their Spherical Harmonics Transform, whose Angular
+Power Spectrum is one the most powerful tools to understand the early Universe.
+See in the ![figure](healpix_grid.png) how a sphere is first split into 12 base
```
Comment (DanielLenz) [inline paper/paper.md]: Start a new paragraph before 'See in the figure...'

File: paper/paper.md [hunk 9]
```diff
@@ -0,0 +1,89 @@
...
+pixels of equal area and whose centers are at the same latitude
+and then each is further subdivided to achieve higher and higher resolution.
+The ``HEALPix`` team provides FORTRAN, C++ and IDL implementations of the framework.
```
Comment (DanielLenz) [inline paper/paper.md]: C++, and IDL

**Comment by zonca** [comment]
thanks @DanielLenz for your feedback!

**Pull request merged**